src/main/java/TextView.java

```diff
@@ -102,6 +113,11 @@ void drawBoard(Board board)
 				}
 			}
 		}
+		System.out.print("  ");
+		for (int i =0; i < board.GRID_SIZE; i++){
+			System.out.print("¯¯¯¯");
+		}
+		System.out.println("  ");
 	}
 
 	/**
```

Comment:
- [ ] General Board Display
    - [ ] The board spacing seems a little off. Each column is a little to the right of the column label.
    - [ ] The top and bottom boundaries of the grid do not extend fully to the right.